cobalt/build/gn.py

```diff
@@ -96,18 +96,24 @@ def write_build_args(build_args_path, original_lines, dict_settings,
 
 
 def main(out_directory: str, platform: str, build_type: str,
-         overwrite_args: bool, gn_gen_args: List[str]):
+         gn_gen_args: List[str]):
   platform_path = f'cobalt/build/configs/{platform}'
   dst_args_gn_file = os.path.join(out_directory, 'args.gn')
   src_args_gn_file = os.path.join(platform_path, 'args.gn')
   Path(out_directory).mkdir(parents=True, exist_ok=True)
 
-  if overwrite_args or not os.path.exists(dst_args_gn_file):
-    build_args = get_build_args(src_args_gn_file)
+  build_args = get_build_args(src_args_gn_file)
+  if not os.path.exists(dst_args_gn_file):
```

Comment:
This should be changed to a positive conditional that renames the old args.gn file if necessary, as the `write_build_args` call is the same in both cases